api/bertydemo.proto

```diff
@@ -4,7 +4,7 @@ package berty.protocol;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
-option go_package = "berty.tech/berty/go/pkg/bertydemo";
+option go_package = "berty.tech/berty/v2/go/pkg/bertydemo";
```

Comment:
nope, you should keep real file system destination

in most of the cases it will work without any hack

the only exception I found was when a .proto file imports another one (and only on current gogoproto compiler version), then you need to apply a `sed` after the `protoc` command (see the current go/Makefile file as an example)